increase min sunpy (#63)
sunpy/sunkit-magex: docs/installing.rst

    pip install sunkit-magex

This will install sunkit_magex and all of its dependencies.

> In addition to the core dependencies, there are optional dependencies (numba) that can improve code performance.

Reviewer: ```suggestion
In addition to the core dependencies, numba is an optional dependency that can improve code performance.
```
